public/javascripts/SVLabel/src/SVLabel/Keyboard.js

```diff
@@ -54,6 +97,24 @@ function Keyboard ($) {
      * @private
      */
     function documentKeyUp (e) {
+        switch (e.keyCode) {
+            // "left"
+            case 37:
+                movePano(270);
+                break;
+            // "up"
+            case 38:
+                movePano(0);
```

Comment:
movePano is not a very descriptive function name... Change it to moveForward and moveBackward?